docs/downloads.md

```diff
@@ -21,10 +21,11 @@ For more information about the license of Qlik Core visit
 
 ## Other Components
 
-In addition, there are several unsupported open source JavaScript libraries that you will find useful when working
-with Qlik Core. You can install these libraries as npm packages:
+In addition, there are several unsupported open source libraries that you will find useful when working
+with Qlik Core. You can install these libraries as npm or golang packages:
```

Comment:
I'd rather we drop the `You can install these libraries as npm or golang packages:` sentence and refer to their GitHub.